GANA-Core.package/CommitOutputSpike.class/instance/showHistoriansInMorph.st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+showHistoriansInMorph
+	| historyWalker sortedRevisions text |
+	historyWalker := SquotHistoryWalker new.
+	sortedRevisions := Heap sortBlock: [:a :b | (a metadata at: #timestamp) > (b metadata at: #timestamp)].
```

Comment:
I refactored that to an instance variable 'byTime' that should kame it clearer -> Heap sortBlock: self byTime